fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/PlanningRuleSet.java

```diff
@@ -173,8 +164,6 @@ public class PlanningRuleSet {
             new ImplementSimpleSelectRule(),
             new ImplementExplodeRule(),
             new ImplementNestedLoopJoinRule(),
-            new PartitionSelectRule(),
```

Comment:
There rules are exploratory rules, not implementation rules. They are moved to `EXPLORATION_RULES`.